driver/event_table.c

```diff
@@ -308,4 +308,6 @@ const struct ppm_event_info g_event_info[PPM_EVENT_MAX] = {
 	/* PPME_SYSCALL_EXECVE_17_X */{"execve", EC_PROCESS, EF_MODIFIES_STATE, 17, {{"res", PT_ERRNO, PF_DEC}, {"exe", PT_CHARBUF, PF_NA}, {"args", PT_BYTEBUF, PF_NA}, {"tid", PT_PID, PF_DEC}, {"pid", PT_PID, PF_DEC}, {"ptid", PT_PID, PF_DEC}, {"cwd", PT_CHARBUF, PF_NA}, {"fdlimit", PT_UINT64, PF_DEC}, {"pgft_maj", PT_UINT64, PF_DEC}, {"pgft_min", PT_UINT64, PF_DEC}, {"vm_size", PT_UINT32, PF_DEC}, {"vm_rss", PT_UINT32, PF_DEC}, {"vm_swap", PT_UINT32, PF_DEC}, {"comm", PT_CHARBUF, PF_NA}, {"cgroups", PT_BYTEBUF, PF_NA}, {"env", PT_BYTEBUF, PF_NA}, {"tty", PT_INT32, PF_DEC} } },
 	/* PPME_SYSCALL_UNSHARE_E */ {"unshare", EC_PROCESS, EF_NONE, 1, {{"flags", PT_FLAGS32, PF_HEX, clone_flags} } },
 	/* PPME_SYSCALL_UNSHARE_X */ {"unshare", EC_PROCESS, EF_NONE, 1, {{"res", PT_ERRNO, PF_DEC} } },
+	/* PPME_SYSCALL_UNLINKAT_3_E */{"unlinkat", EC_FILE, EF_NONE, 3, {{"dirfd", PT_FD, PF_DEC}, {"name", PT_CHARBUF, PF_NA}, {"flags", PT_FLAGS8, PF_HEX, unlinkat_flags} } },
```

Comment:
Why the 3 here, and not UNLINKAT_2_{E,X}?

Reply:
As far as I understand the "version" represent the number of parameters that the syscall represent.
Take `execve` for example, the first "version" is `PPME_SYSCALL_EXECVE_8_{E,X}` that handles 8 parameters, then the next version is `PPME_SYSCALL_EXECVE_13_{E,X}` that handles 13 parameters... and so on.
If I misunderstood convention this I can change it to `UNLINKAT_2_{E,X}`.

Reply:
3 is just an incremental version number, it does not represent argument count

Reply:
Changed to PPME_SYSCALL_UNLINKAT_2_{E/X}